src/test/java/uk/gov/companieshouse/filinghistory/api/controller/AnnotationTransactionIT.java

```diff
@@ -110,7 +109,11 @@ void shouldAddAnnotationToDocumentAndReturn200OK() throws Exception {
                 .replaceAll("<barcode>", BARCODE)
                 .replaceAll("<transaction_id>", TRANSACTION_ID)
                 .replaceAll("<company_number>", COMPANY_NUMBER)
-                .replaceAll("<parent_entity_id>", ENTITY_ID);
+                .replaceAll("<parent_entity_id>", ENTITY_ID)
+                .replaceAll("<updated_at>", UPDATED_AT.toString())
+                .replaceAll("<updated_by>", CONTEXT_ID)
```

Comment:
Is there a benefit of setting updated_by and created_by every time? They will never change in the tests. Same goes for the other integration tests.

Reply:
created_by should be different to updated_by in a couple of the tests just to make sure the code could handle different strings I believe. But they're not massively different so I'm happy to keep them the same and remove the line if you like. Thought it was better for clarity too to have them all set in the tests if there was a possibility they would change in future.

Reply:
We should only make the field variable in the tests it needs to change in. The tests are already quite hard to read so the fewer lines the better.

Reply:
Okay cool will change

Reply:
Done